framework/data/Sort.php

```diff
@@ -438,14 +438,25 @@ public function createSortParam($attribute)
         $definition = $this->attributes[$attribute];
         $directions = $this->getAttributeOrders();
         if (isset($directions[$attribute])) {
-            $direction = $directions[$attribute] === SORT_DESC ? SORT_ASC : SORT_DESC;
+            if ($this->enableMultiSort && count($directions) > 1) {
```

Comment:
I think it would be nice to have this feature when `enableMultiSort` is on even for one sortable column.